getting-started-github-graphql-api.md

```diff
@@ -40,59 +42,59 @@ When interacting with GitHub's GraphQL API, it would be great to fill your own a
 
-Whereas GitHub's GraphQL Explorer knows about your credentials, since you need to sign up using it, the GraphiQL application needs to know about your personal access token that you have created before. You can add the generated access token in your HTTP header for every request by opening up the headers configuration.
+GitHub's GraphQL Explorer knows about your credentials, since you need to sign up using it, but the GraphiQL application needs to know about the personal access token you created. You can add it in your HTTP header for every request inthe headers configuration.
 
 {{% pin_it_image "graphiql headers" "img/posts/getting-started-github-graphql-api/graphiql-headers.jpg" "is-src-set" %}}
 
-In the next step, you need to add a new header with a name and value to your GraphiQL configuration. In order to communicate with GitHub's GraphQL API, you need to fill in the header name with "Authorization" and the header value with "bearer [your personal access token]". Afterward, save this new header for your GraphiQL application. Finally, you are ready to make requests to GitHub's GraphQL API with your GraphiQL application.
+In the next step, we add a new header with a name and value to your GraphiQL configuration. To communicate with GitHub's GraphQL API, fill in the header name with "Authorization" and the header value with "bearer [your personal access token]". Save this new header for your GraphiQL application. Finally, you are ready to make requests to GitHub's GraphQL API with your GraphiQL application.
 
 {{% pin_it_image "graphiql authorization" "img/posts/getting-started-github-graphql-api/graphiql-authorization.jpg" "is-src-set" %}}
 
-Last but not least, in case you end up using your own GraphiQL application, you need to provide the GraphQL endpoint for GitHub's GraphQL API which is `https://api.github.com/graphql`. In case of GitHub's GraphQL API, you are going to use the {{% a_blank "POST HTTP method" "https://en.wikipedia.org/wiki/Hypertext_Transfer_Protocol#Request_methods" %}} for GraphQL queries and mutations. After all, you have to transfer the data as payload to your GraphQL endpoint.
+If you use your own GraphiQL application, you'll need to provide the GraphQL endpoint for GitHub's GraphQL API:`https://api.github.com/graphql`. For GitHub's GraphQL API, use the {{% a_blank "POST HTTP method" "https://en.wikipedia.org/wiki/Hypertext_Transfer_Protocol#Request_methods" %}} for queries and mutations, and to transfer data as a payload to your GraphQL endpoint.
```

Comment:
... GitHub's GraphQL API:`https://api.github.com/graphql` <- space in between

Reply:
Space in between where?

Reply:
I think after the : there is no space.